### DISCUSSION TITLE
Event listener for repeatable jobs

### DISCUSSION THREAD

**Discussion by gee4vee** [taskforcesh/bullmq#2709] [Q&A]
If I create a job with a cron repeat strategy, is there a way to be notified when the next repeat job is added to the queue? I would like to act on that to create things in my application as long as the repeatable jobs are being added.

**Comment by roggervalf** [comment]
hi @gee4vee, we have a delayed event https://api.docs.bullmq.io/interfaces/v5.QueueEventsListener.html#delayed, so you may need to get the job instance from that jobId to validate if a job is an instance of a repeatable job

**Comment by gee4vee** [comment]
@roggervalf Are there any guarantees about the order of this event with respect to the job being picked up by a worker? I would like the event to be processed by my application before the worker starts processing the next job.
